vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -41,12 +43,18 @@ def get_pods(inventory)
 
     def get_endpoints(inventory)
       process_collection(inventory["endpoint"], :container_endpoints) { |n| parse_endpoint(n) }
+
       @data[:container_endpoints].each do |ep|
         @data_index.store_path(:container_endpoints, :by_namespace_and_name,
                                ep[:namespace], ep[:name], ep)
       end
     end
 
+    def get_replication_controllers(inventory)
+      process_collection(inventory["replication_controller"], :container_replication_controllers)\
+                                                                                { |n| parse_replication_controllers(n) }
```

Comment:
@Fryguy I warned you (here: https://github.com/ManageIQ/manageiq/pull/2390#discussion_r27647864) that trying to maintain meaningful iterator names is hard, I guess now instead of having `x` we'll have `n` because we're always copying-pasting from the first method `get_nodes` and we forget to enforce the meaningful name (e.g. `rc`).